src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/MemberDeclarationSyntaxExtensions.cs

```diff
@@ -87,17 +89,20 @@ public static int GetArity(this MemberDeclarationSyntax member)
         return 0;
     }
 
-    public static TypeParameterListSyntax GetTypeParameterList(this MemberDeclarationSyntax member)
+#nullable enable
```

Comment:
I assume it's a fair bit of work just to do this for the whole file?

Reply:
yes.  i tried, and it was a PITA :)